lib/assert.js

```diff
@@ -151,6 +154,16 @@ x.ifError = (err, msg) => {
 	test(!err, create(err, 'Error', '!==', msg, x.ifError));
 };
 
+x.jsxEqual = (val, expected, msg) => {
+	const treeEquals = reactElementToJSXString(val) === reactElementToJSXString(expected);
+	test(treeEquals, create(val, expected, '===', msg, x.jsxEqual));
+};
+
+x.jsxNotEqual = (val, expected, msg) => {
```

Comment:
I was changing it back and forth, probably should be that way indeed. I didn't like capitalization of `jsx` is different in `jsxEqual` and `notJsxEqual`.